docs/user-guide/work-packages/exporting/README.md

```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
 
 
 
-## Changing what will be exported
+## Export contents
 
-From the work packages module, all work packages that are included in the filter table will be exported. Up to a total of 500 work packages can be exported at once.
+All work packages that are included in the work package table in the currently selected view will be exported, unless a certain export limit has been defined by the instance administrator. The limit can be changed in the [work package settings](../../../system-admin-guide/system-settings/general-settings/#general-system-settings) in the system administration. Newly created instances have a maximum of 500 work packages set as a limit by default.
 
-> **Note**: If you need to export more than 500 items, filter the work packages appropriately to export multiple files.
+> **Note**: PDF export options includes all of the work packages in the selected work package table, regardless of the limit. The possible export limit is relevant for XLS, CSV and Atom export options.
 
-For self-hosted versions of OpenProject you can change this limit in the [work package settings](../../../system-admin-guide/manage-work-packages/work-package-settings/) in the system administration.
+**Columns**
 
+The exported file will display the columns that are activated for the work package table. By adding or removing specific columns you can control the columns that will be included into the exported file. Please make sure to [save](../work-package-table-configuration/#save-work-package-views) the work package view you configured for the changes to be included into the report.
 
-**Columns**
+Some formats such as PDF will limit the number of columns available due to limitations of the PDF rendering engine to avoid overflowing the available space. 
 
-The exported file will display the columns that you activated for the work package table. With it, you can control the columns that will be contained in various formats. Please make sure to [save](../work-package-table-configuration/#save-work-package-views) the view you configured, as saving is a necessary precondition to keep the selected columns in the export.
 
-Some formats such as PDF will limit the number of columns available due to limitations of the PDF rendering engine to avoid overflowing the available space.
 
+## Export format options
+### PDF export
 
+OpenProject has multiple options of exporting work packages in PDF format:
 
-## PDF export
+- **PDF Table** exports the work package table displaying work packages as single rows.
 
-OpenProject has multiple options of exporting the results to PDF:
+![OpenProject PDF Table export](openproject_pdf_table_export.png)
 
+- **PDF Report** includes a table of contents (listing all of the work packages), followed by the description of single work packages in a block form. For each work package a table of attributes is included (attributes correspond to the columns in the work package table). 
 
```

Comment:
Add a small introduction, e.g. : With PDF Reports you can export detailed up to date work plans for your project in a nice format. ...It includes ...